tests/foreman/cli/test_repository.py

```diff
@@ -229,7 +262,7 @@ def test_positive_create_with_auth_yum_repo(self):
         url = FAKE_5_YUM_REPO
         for creds in valid_http_credentials(url_encoded=True):
             url_encoded = url.format(creds['login'], creds['pass'])
-            with self.subTest(url):
+            with self.subTest(creds):
```

Comment:
cred is dict perhaps url_encoded should be more appropriate?

Reply:
but here
https://github.com/SatelliteQE/robottelo/pull/5691/files#diff-c5b3908e7803d076d19a827bf7c96c31R697
it is fine.
what is right way?